commerce/domain/tax/service/classes/TaxServiceExtensionSampleForCanada.cls

```diff
@@ -0,0 +1,13 @@
+// This is a sample placeholder for taxation Extension Provider for en_CA locale
+// Please refer the TaxServiceExtensionSample sample class for a detailed custom implementation of an Extension Provider.
+public class TaxServiceExtensionSampleForCanada extends commercestoretax.TaxService {
```

Comment:
Does this class needs to be on public repo? If yes, may be we need to add a better sample with comments or clone TaxServiceExtensionSample class with little modifications for Canada.

Reply:
For Canada we have just added the placeholder and the idea of including Canada was to show that multiple extension providers can be used and the customer can refer to TaxServiceExtensionSample for implementing the ExtensionProviders. We have mentioned this in comments on the top of Canada's ExtensionProvider as well. This was finalised after a discussion with Santosh. Same we have checked in in Core.

Reply:
If needed we can remove this placeholder apex from public repo as we are nowhere referring this in the resolver apex.